### ISSUE TITLE
{name}.sub.{ext} server-side substitution of domains broken on w3c-test.org

### ISSUE THREAD

**Issue by foolip** [w3c/web-platform-tests#10236]
This is about the feature documented on http://web-platform-tests.org/writing-tests/server-features.html as served on http://w3c-test.org

https://github.com/w3c/web-platform-tests/blob/master/generic-sensor/generic-sensor-feature-policy-test.sub.js has as input:
```JavaScript
const cross_origin_src =
  "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
```

But https://w3c-test.org/generic-sensor/generic-sensor-feature-policy-test.sub.js has in its output:
```JavaScript
const cross_origin_src =
  "https://www.web-platform.test:443" + same_origin_src;
```

Some configuration should have made that "https://www.w3c-test.org:443" instead.

@gsnedders @sideshowbarker, do either of you know what might be causing this?

**Comment by mdittmer** [comment]
@foolip WDYT the priority should be here? I don't have enough context to make the call myself.

**Comment by foolip** [comment]
@mdittmer, let's call it roadmap, because it affects lots of tests and ought to be a fairly simple fix.

**Comment by sideshowbarker** [comment]
Fixed.

The config file had the `host` key set the right value, but I guess somebody (without giving me any heads-up) at some point changed the wptserve code to now expect a `browser_host` key instead? If so, that makes me wonder if there are other changes like that which I also might never have gotten a heads-up about

**Comment by gsnedders** [comment]
@sideshowbarker it should still work with host? um. that sounds like a bug.

**Comment by gsnedders** [comment]
(maybe if the original config file has `browser_host` and the override has `host` it's broken? anyway, if it is broken then that'll be fixed in #10231 which I'm confident is right then)

**Comment by foolip** [comment]
Thanks for fixing, @sideshowbarker!
